action/protocol/execution/evm/evm.go

```diff
@@ -292,7 +293,7 @@ func ExecuteContract(
 		data := retval[4:]
 		msgLength := byteutil.BytesToUint64BigEndian(data[56:64])
 		revertMsg := string(data[64 : 64+msgLength])
-		receipt.SetExecutionRevertMsg(revertMsg)
+		receipt.SetExecutionRevertMsg(fmt.Sprintf("execution reverted: %s", revertMsg))
```

Comment:
It's better to insert "execution reverted:" at api/web3server.go instead of evm.go, leading to much unnecessary changes

Reply:
don't change it in EVM, which will cause a HF